pyproject.toml

```diff
@@ -203,13 +202,6 @@ show_error_codes = true
 # strict_equality = true
 # # --strict end
 
-# black code formatting
-# https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file
-[tool.black]
-line-length = 100
```

Comment:
is the default line-length for ruff 100? I don't see this configured below

Reply:
The default is 88, but we'd previously specified this as 100 under `tool.ruff` which is why you don't see a diff there.